mlflow/utils/search_utils.py

```diff
@@ -957,3 +1052,140 @@ def _invalid_statement_token_search_model_registry(cls, token):
             return False
         else:
             return True
+
+
+class SearchModelVersionUtils(SearchUtils):
+    NUMERIC_ATTRIBUTES = {"creation_timestamp", "last_updated_timestamp"}
+    # what should we support here?
+    VALID_SEARCH_ATTRIBUTE_KEYS = {
+        "name",
+        "run_id",
+        "source_path",  # this is for sql store only
```

Comment:
Can we support it for file store as well?

Reply:
Supporting search by source has some conflicts with existing methods, as `source` is identified as a TokenType.Keyword and the value is identified as TokenType.Literal.String.Single. I'll investigate more on this :)